tests/unit/common/test_lambda_handler.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+
+from botocore.stub import Stubber
+
+from matrix.common.lambda_handler import LambdaHandler
+from matrix.common.lambda_handler import LambdaName
+
+
+class TestLambdaHandler(unittest.TestCase):
+    def setUp(self):
+        self.handler = LambdaHandler()
+        self.mock_handler = Stubber(self.handler._client)
+
+    def test_invoke(self):
+        expected_params = {'FunctionName': LambdaName.DRIVER.value,
+                           'InvocationType': "Event",
+                           'Payload': b"{}"}
+        self.mock_handler.add_response('invoke', {}, expected_params)
+        self.mock_handler.activate()
+        self.handler.invoke(LambdaName.DRIVER, {})
```

Comment:
This looks to be causing issues in the travis test build. Are you sure its mocked or working as you intended here?

Reply:
Environment variable issue